jbehave-support-core/src/main/java/org/jbehavesupport/core/internal/verification/VerifierResolverImpl.java

```diff
@@ -26,4 +27,12 @@ public Verifier getVerifierByName(String name) {
             .orElseThrow(() -> new NoSuchBeanDefinitionException(name));
     }
 
+    public Verifier getVerifierByName(String name, String defaultVerifierName) {
+        if (!isEmpty(name)) {
+            return getVerifierByName(name);
```

Comment:
Method throws an exception if none or more verifiers is found, shouldn't it fallback to default in that case?

Reply:
Kinda makes sense, but also would theoretically break backwards compatibility (tests that were failing before could potentially start working for bad configuration or if user would input nonsensical verifier name, which could be misleading if it works)

Reply:
Discussed in call - behavior is fine with just checking the name and not catching exceptions.